skimage/measure/tests/test_marching_cubes.py

```diff
@@ -30,21 +30,29 @@ def test_marching_cubes_anisotropic():
     ellipsoid_anisotropic = ellipsoid(6, 10, 16, spacing=spacing,
                                       levelset=True)
     _, surf = ellipsoid_stats(6, 10, 16)
-    
+
     # Classic
     verts, faces = marching_cubes_classic(ellipsoid_anisotropic, 0.,
                                           spacing=spacing)
     surf_calc = mesh_surface_area(verts, faces)
     # Test within 1.5% tolerance for anisotropic. Will always underestimate.
     assert surf > surf_calc and surf_calc > surf * 0.985
-    
+
     # Lewiner
     verts, faces = marching_cubes_lewiner(
         ellipsoid_anisotropic, 0., spacing=spacing)[:2]
     surf_calc = mesh_surface_area(verts, faces)
     # Test within 1.5% tolerance for anisotropic. Will always underestimate.
     assert surf > surf_calc and surf_calc > surf * 0.985
 
+    # Test marching cube with mask
+    with pytest.raises(AttributeError):
```

Comment:
Now it's a `ValueError` ;-)

Reply:
I had not seen that test, thank for the catch! I just sent the fix, lets hope this pass now